### PR TITLE
revert 9416

### PR DESCRIPTION
**Pull request by mairaw**
Revert PR #9416

### CODE REVIEW

**Comment by mairaw** [comment]
@rpetrusha you mentioned we have our own Dispose guidance, which will redirect this to. Maybe we should investigate whether @migueldeicaza's contributions are already part of that topic.

**Review by rpetrusha — APPROVED** [review]
LGTM, @mairaw. There's one case of an article ("a") that was omitted. I'm not sure whether it can or should be added, though. You can merge when you're ready.

File: docs/standard/design-guidelines/dispose-pattern.md [hunk 1]
```diff
@@ -184,12 +184,10 @@ public class Stream : IDisposable {
   
 <a name="finalizable_types"></a>   
 ## Finalizable Types  
- Finalizable types are types that extend the Basic Dispose Pattern by overriding the finalizer and providing finalization code path in the `Dispose(bool)` method. Usually they are implemented to properly release unmanaged resources.
+ Finalizable types are types that extend the Basic Dispose Pattern by overriding the finalizer and providing finalization code path in the `Dispose(bool)` method.
```
[the remark this answers is not in the document]
Comment (mairaw) [reply docs/standard/design-guidelines/dispose-pattern.md]: I really don't know what we're allowed to touch or not beyond cosmetic changes

**Comment by migueldeicaza** [comment]
Hey folks - what is the rationale to revert this patch?

**Comment by mairaw** [comment]
Hi @migueldeicaza, I left a note on the original PR (https://github.com/dotnet/docs/pull/9416#issuecomment-444651644). Basically, we can't touch this content because it's copyrighted material.

**Pull request merged**